client/src/App.tsx

```diff
@@ -11,20 +11,29 @@ import { ThemeProvider } from '@fluentui/react';
 import Telemetry from './components/telemetry';
 
 export const App: FC = () => {
+  const [listId, setListId] = useState("")
```

Comment:
What's this useState for?

Reply:
It's a global getter and setter for current selected list it, when this value changed, it will effect other dependencies, keep updated data in sync.